README.md

```diff
@@ -52,6 +52,15 @@ Install the following extensions:
 
 ## Code scaffolding
 
+## Commit format
```

Comment:
Your text block is hurting the "code scaffolding" section please solve it.
I think your text block would fit better as another option of the menu "Prepare your environment". 
Please change "Commit format" to "Commit message format"